src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -1053,10 +1053,12 @@
 			v = "AdvancedVectorExtensions"
 		elseif x == "AVX2" and _ACTION > "vs2012" then
 			v = "AdvancedVectorExtensions2"
-		elseif x == "SSE2" then
-			v = "StreamingSIMDExtensions2"
-		elseif x == "SSE" then
-			v = "StreamingSIMDExtensions"
+		elseif cfg.architecture ~= "x64" then
```

Comment:
Well, I can certainly rename the if to x86_64.... I think x86 would be wrong however, since we still want this enabled on 32-bit platforms, where this feature is not implied... old x86 processors don't always have SSE2, although you have to go back in history quite a bit for that you still have to explicitly enable this in Visual Studio.